src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -81,6 +81,11 @@ void WorldSession::HandleGroupInviteOpcode(WorldPacket& recvData)
         return;
     }
 
+    if (GetPlayer()->IsGameMaster() && !GetPlayer()->IsAcceptingInvites())
```

Comment:
You could add this to the check above to also sendERR_INVITE_RESTRICTED

Reply:
@Kitzunu The ERR_INVITE_RESTRICTED message is

```
Trial accounts cannot invite characters into groups.
```

Reply:
ERR_BAD_PLAYER_NAME_S Should be fine